runhouse/servers/http/http_server.py

```diff
@@ -1253,6 +1269,10 @@ async def main():
         if env_vars:
             cluster.call(cluster.default_env.name, "_set_env_vars", env_vars)
 
+    log_level = parse_args.log_level or DEFAULT_LOG_LEVEL
```

Comment:
do we need the "or" here? won't it have the default value anyway if not explicitly provided?